### ISSUE TITLE
Create a11y evaluation tools index

### ISSUE THREAD

**Issue by amycole501** [uswds/uswds#5340]
Create an accessibility audit inventory of resources including AMP, Axe dev tools, etc., and identify missing features we need.

 This ticket will be considered done when a working draft is complete.

**Comment by amycole501** [comment]
https://insite.gsa.gov/employee-resources/information-technology/it-accessibility-section-508/accessibility-solutions/accessibility-management-platform-amp?check_logged_in=1

**Comment by amycole501** [comment]
Current working inventory https://docs.google.com/spreadsheets/d/1_a7zTnx8wmvzqCDqf7wenaeS5a2Rj2BQiAfu6Lew8G8/edit#gid=1680581190

**Comment by amycole501** [comment]
I created an inventory spreadsheet and anyone can add to it. I included a column for frequency of use. This is a "living document" for future contribution.

**Comment by brunerae** [comment]
Closing as complete.
